src/Stratis.Bitcoin.Features.BlockStore/Models/BlockVerboseModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Linq;
+using NBitcoin;
+using Stratis.Bitcoin.Controllers.Models;
+
+namespace Stratis.Bitcoin.Features.BlockStore.Models
+{
+    public class BlockVerboseModel : BlockModel
```

Comment:
Maybe rename to `BlockTransactionDetailsModel` - as this is what the model is doing.  The term verbose could mean other details then txns within a `Block`.

Reply:
Thanks, fixed in the coming update. I named it to be same as TransactionVerboseModel, which existed from before. Has been renamed to BlockTransactionDetailsModel.